src/main/scala/cromwell/engine/backend/BackendCall.scala

```diff
@@ -63,9 +63,14 @@ final case class SuccessfulExecutionHandle(outputs: CallOutputs, events: Seq[Exe
   override val result = SuccessfulBackendCallExecution(outputs, events, returnCode, hash, resultsClonedFrom)
 }
 
-final case class FailedExecutionHandle(throwable: Throwable, returnCode: Option[Int] = None) extends ExecutionHandle {
+final case class FailedExecutionHandle(throwable: Throwable, returnCode: Option[Int] = None, events: Seq[ExecutionEventEntry] = Seq.empty) extends ExecutionHandle {
```

Comment:
I think as-is it should work already, it'll just be on a separate row... as long as all of the failed attempts still show up in metadata, we should be fine.